src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -944,17 +937,33 @@ func (c controller) waitForCSV(ctx context.Context, waitTimeout time.Duration) e
 	return utils.WaitForPredicateWithContext(ctx, waitTimeout, GeneralWaitInterval, areOLMOperatorsAvailable)
 }
 
-// validateConsoleAvailability checks if the console operator is available
-func (c controller) validateConsoleAvailability() bool {
-	return c.isOperatorAvailable(NewClusterOperatorHandler(c.kc, consoleOperatorName))
+// waitForClusterOperators checks Console operator and the Cluster Version Operator availability in the
+// new OCP cluster.
+// A success would be announced only when the service acknowledges the operators availability,
+// in order to avoid unsycned scenarios.
+func (c controller) waitForClusterOperators(ctx context.Context) error {
+	operatorsErrGroup, _ := errgroup.WithContext(ctx)
+
+	operatorsErrGroup.Go(func() error {
+		validateConsoleAvailability := func() bool {
+			return c.isOperatorAvailable(NewClusterOperatorHandler(c.kc, consoleOperatorName))
+		}
+		return utils.WaitForPredicateWithContext(ctx, WaitTimeout, GeneralWaitInterval, validateConsoleAvailability)
```

Comment:
in my opinion we shouldn't have timeout for console here. Lets have only CVO timeout and console should run till CVO us running

Reply:
It was designed that each operator has a different timeout. Do you want to change it? In your comment you suggest to keep running the console validation regardless of success, until CVo finishes

Reply:
CVO will not finish in case console will not be available.
CVO has a logic that restarts timeout in case of different message but console doesn't.
In case cvo will continue but console still will not be available cause for example AUTH is not ready yet we will fail on console wait but cvo will still be in valid case

Reply:
Okay.
Do you wish to keep the `waitForClusterVersion` configurable? In that case of always depending on CVO to run the console, we would ignore validating the `console`. Or do you wish to keep running `console` in case CVO validation is turned off?

Reply:
good question, if we leave waitForClusterVersion configurable , we must wait for console